roles/container-engine/containerd/tasks/main.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: containerd | Install required packages
+  apt:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - libseccomp2
+    - ethtool
+
+- name: containerd | Download files
+  include_role:
+    name: download
+  vars:
+    download_container: false
+
+- name: containerd | Unpack release tarball
+  unarchive:
+    src: "{{ local_release_dir }}/cri-containerd-cni-{{ containerd_version }}.linux-amd64.tar.gz"
```

Comment:
Wrong way to do it. Should be in the download role as a binary download.

Reply:
Just for clarification, you would like the "containerd | Download files" step to be removed from this role? It is already utilising the download role to do the downloading.
The reason for having the step in this role is because in cluster.yml the download role is applied after the container-engine role thus making the binaries not being in place when they are needed. Moving the container engine role after the download role in cluster.yml would lead to being unable to pull container images due to missing a container engine.